media-sound/denemo/files/denemo-2.2.0-no-portaudio-fix.patch

```diff
@@ -0,0 +1,29 @@
+From 2e3c3b974e3e27873841c0ad9a05537a682e85f5 Mon Sep 17 00:00:00 2001 
```

Comment:
Was this accepted upstream?

Reply:
Yes, it looks like it is: http://git.savannah.gnu.org/cgit/denemo.git/commit/?id=2e3c3b974e3e27873841c0ad9a05537a682e85f5
Also, when I download the master branch, it's in `src/audio/midi.c`.